website/src/pages/admin/trollboard.tsx

```diff
@@ -22,6 +39,16 @@ const Leaderboard = () => {
           </Heading>
           <Card>
             <CardBody>
+              <RadioGroup defaultValue="1" onChange={setEnabled.toggle}>
+                <Stack direction="row" spacing={5}>
+                  <Radio value="1" colorScheme="green">
+                    {t("show_active_user")}
+                  </Radio>
+                  <Radio value="2" colorScheme="red">
+                    {t("show_banned_user")}
+                  </Radio>
+                </Stack>
+              </RadioGroup>
```

Comment:
Maybe a checkbox is enough herer?

Reply:
It would be enough, but since the behavior of the trollboard is changing due this filter I wanted to make it crystal clear, now you only see active users or banned users. If I did it with a checkbox, the user might expect to see all users (active + banned, like it works right now), and check to only see active ones. But that's not the way it behaves with this new "enabled" flag

Reply:
This can be replaced by a select with 3 option which is more flexible
- Only active users (enabled = true)
- Only banned users (enabled = false)
- All users (enabled = null)